SimPEG/Maps.py

```diff
@@ -1871,9 +1893,306 @@ def deriv(self, m):
             ]).T)
 
 
-class ParametrizedCasingAndLayer(ParametrizedLayer):
+class ParametricBlock(BaseParametric):
+    """
+        Parametric Block in a Layered Space
```

Comment:
is this in layered earth? 
I thought you are assuming homogenous earth (or background is layered-earth here?)

Reply:
ah, thanks that is an old doc-string that I copied from the Parametric layer map